client/components/domains/domain-product-price/index.jsx

```diff
@@ -15,12 +18,17 @@ const DomainProductPrice = React.createClass( {
 		requiresPlan: React.PropTypes.bool
 	},
 	renderFreeWithPlan() {
+		const product_price = ! domainsWithPlansOnlyTestEnabled &&
+							<span
+								lassName="domain-product-price__price">{ this.translate( '%(cost)s {{small}}/year{{/small}}', {
+									args: { cost: this.props.price },
+									components: { small: <small /> }
+								} ) }</span>;
+
 		return (
-			<div className="domain-product-price is-free-domain">
-				<span className="domain-product-price__price">{ this.translate( '%(cost)s {{small}}/year{{/small}}', {
-					args: { cost: this.props.price },
-					components: { small: <small /> }
-				} ) }</span>
+			<div
+				className={ 'domain-product-price is-free-domain' + ( domainsWithPlansOnlyTestEnabled && ' no-price' ) }>
```

Comment:
Ouch x2 - another reason to use `classnames`: `( domainsWithPlansOnlyTestEnabled && ' no-price' )` evaluates to `false`... which, in this context, is cast to a string, so you end up with these classes: `domain-product-price is-free-domainfalse` (notice the `false` at the end 😅 ).